lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -25,6 +25,31 @@ local os_sep = Path.path.sep
 -- enclose in module for docgen
 local fb_picker = {}
 
+-- try to get the index of entry of current buffer
+local get_selection_index = function(opts, results)
+  local init_dir = Path:new(opts.path):absolute()
```

Comment:
I'd expect `opt.path` to be an absolute path here at last, since it's already been `vim.fn.expand`ed.